meson.build

```diff
@@ -2903,6 +2903,25 @@ test('tmp_install',
     is_parallel: false,
     suite: ['setup'])
 
+# it seems freebsd doesn't use libdir for pkgconfig path
+if host_system == 'freebsd'
+  pkgconf_installdir =  dir_prefix / 'libdata' / 'pkgconfig'
+else
+  pkgconf_installdir = dir_prefix / dir_lib / 'pkgconfig'
+endif
+test_pkg_conf_file = files('src/tools/ci/test_pkg_conf')
+test('pkg_conf_extensions',
+    test_pkg_conf_file,
+    args: [
+      '--rootdir', meson.source_root(),
+      '--pkgconf_installdir', pkgconf_installdir,
+      '--builddir', meson.build_root(),
+      '--pkg_conf_path', get_option('pkg_config_path'),
+      '--meson', meson_bin.path(),
```

Comment:
I don't think that's quite right - meson_bin often just be 'python', needs to be combined with meson_args.